### DISCUSSION TITLE
Unknown Error Occurred

### DISCUSSION THREAD

**Discussion by cassidyt** [GorvGoyl/ChatGPT-Writer#47] [Report Bugs and Issues]
Hi, 

I downloaded the extension and have been unable to use it. It consistently spits out the following message. I'm able to open the extension in Gmail and tyle in it, but it give the following error message when I go to generate the response.  

"There was an unknown error. Make sure you are logged-in to chat.openai.com
If issue persists, please report it here." 

Note: I am on ChatGPT Plus, so that may have something to do with it.

**Comment by grrvin** [comment]
Same as cassidyt. Plus member as well. I have also uninstalled the extension and reinstalled it again. I get the same error.

Current version of Chrome: Version 109.0.5414.119 (Official Build) (x86_64)
Current OS: macOS 13.1 (22C65)

**Reply by grrvin** [reply]
Yep. Working for me now.

**Comment by basitf** [comment]
I am seeing this issue as well once I updated to plus.

**Comment by friuns2** [comment]
same here updated to plus and stop work

**Comment by jxduf** [comment]
I get this error even tho I am logged and I got ChatGPT Plus
There was an unknown error. Make sure you are logged-in to [chat.openai.com](https://chat.openai.com/)
If issue persists, please [report it here](https://github.com/GorvGoyl/ChatGPT-Writer/discussions/categories/report-bugs-and-issues).

**Comment by DarrenRobinson777** [comment]
I also get the same error, even though I am logged in to ChatGPT Plus.

**Reply by DarrenRobinson777** [reply]
Working for me now. Thanks for the quick fix.

**Comment by GorvGoyl** [comment]
Thanks for reporting. I've fixed the issue in v2.1.9. It should be live within a day and then you can update the extension or restart the browser.

https://chrome.google.com/webstore/detail/chatgpt-writer-email-writ/pdnenlnelpdomajfejgapbdpmjkfpjkp/

**Reply by grrvin** [reply]
Yep. Working for me now.

**Comment by jxduf** [comment]
Works for me! Thank you very much!


*Jean-Xavier Dufresne*

Courtier immobilier résidentiel

Focus Immobilier

***@***.***

(819) 213-2586
